src/server/collections/usersDatabase.js

```diff
@@ -164,6 +164,30 @@ function findUsername(id, callback) {
   });
 }
 
+function findUsers(idArray, callback) {
+  MongoClient.connect(url, (err, db) => {
+    if (err !== null) {
+      console.log('[MONGO ERROR] Unable to connect to db: ', err);
+      return;
+    }
+
+    let newIdArray = idArray.map(function(id) {
+      let newId = new ObjectId(id);
```

Comment:
you dont need to declare a variable for this, you can return the new ObjectId(id) directly